app/src/main/res/layout/fragment_device_list2.xml

```diff
@@ -18,9 +18,20 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:scrollbars="vertical"/>
-
     </android.support.v4.widget.SwipeRefreshLayout>
 
+    <android.support.v7.widget.AppCompatTextView
+        android:id="@+id/empty_message"
+        fontPath="@string/normal_text_font_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:gravity="center"
+        android:text="No devices found.\nPress '+' to add your device."
```

Comment:
I probably did this in the past too, but from here out, let's only use resources for strings, instead of hard coding them.